src/array.js

```diff
@@ -49,10 +49,13 @@ TransisArray.of = function() {
 // Public: Creates a new `Transis.Array` from the given array-like object. Useful for converting a
 // regular array to a `Transis.Array`.
 //
-// a - An array-like object.
+// a     - An array-like object.
+// mapFn - Map function to call on every element of the array (optional).
 //
 // Returns a new `Transis.Array`.
-TransisArray.from = function(a) { return TransisArray.of.apply(null, a); };
+TransisArray.from = function(a, mapFn) {
+  return TransisArray.of.apply(null, mapFn ? a.map(a => mapFn(a)) : a);
```

Comment:
Also, the native `Array.from` method accepts a `thisArg` as well to use within the `mapFn`:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from